spec.html

```diff
@@ -9274,7 +9274,7 @@ <h1>Contains</h1>
     <emu-clause id="sec-static-semantics-contains" oldids="sec-object-initializer-static-semantics-contains,sec-static-semantics-static-semantics-contains,sec-function-definitions-static-semantics-contains,sec-arrow-function-definitions-static-semantics-contains,sec-generator-function-definitions-static-semantics-contains,sec-async-generator-function-definitions-static-semantics-contains,sec-class-definitions-static-semantics-contains,sec-async-function-definitions-static-semantics-Contains,sec-async-arrow-function-definitions-static-semantics-Contains" type="sdo">
       <h1>
         Static Semantics: Contains (
-          _symbol_: unknown,
+          _symbol_: a grammar symbol,
```

Comment:
is "a grammar symbol" something defined already?

Reply:
It's a term that the spec uses already, but (as with most grammar-related terms) it's not defined with a `<dfn>`.